<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>客户端</string>
  </property>
  <property name="styleSheet">
   <string notr="true">MainWindow{
background-color:#EEEEEE
}
QPalette{background:#CFDDEE;}QGroupBox#gboxDevicePanel&gt;QLabel{color:#324C6C;}

QWidget#frmMain,QWidget[Form=&quot;true&quot;]{
border:1px solid #B4C2D7;
border-radius:0px;
}

.QFrame{
border:1px solid #B4C2D7;
border-radius:5px;
}

QLabel,QLineEdit,QTextEdit,QPlainTextEdit,QSpinBox,QGroupBox,QComboBox,QDateEdit,QTimeEdit,QDateTimeEdit,QSpinBox,QTreeView,QListView,QTableView,QTabWidget::pane{
color:#324C6C;
}

QWidget#widget_title{
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #C0D3EB,stop:1 #BCCFE7);
}

QLabel#lab_Ico,QLabel#lab_Title{
border-radius:0px;
color:#324C6C;
background-color:rgba(0,0,0,0);
border-style:none;
}

QToolButton::menu-indicator{
image:None;
}

QToolButton,QWidget#widget_frm&gt;QLabel{
border-style:none;
padding:10px;
color:#324C6C;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #C0D3EB,stop:1 #BCCFE7);
}

QToolButton:hover,QWidget#widget_frm&gt;QLabel:hover{
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #D2E3F5,stop:1 #CADDF3);
}

QLabel[labVideo=&quot;true&quot;]{
color:#324C6C;
border:1px solid #B4C2D7;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #C0D3EB,stop:1 #BCCFE7);
}

QLabel[labVideo=&quot;true&quot;]:focus{
border:1px solid #FF0000;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #D2E3F5,stop:1 #CADDF3);
}

QLineEdit,QTextEdit,QPlainTextEdit,QSpinBox{
border:1px solid #B4C2D7;
border-radius:5px;
padding:2px;
background:none;
selection-background-color:#C0D3EB;
selection-color:#324C6C;
}

QLineEdit[echoMode=&quot;2&quot;]{
lineedit-password-character:9679;
}

.QGroupBox{
border:1px solid #B4C2D7;
border-radius:5px;
}

.QPushButton{
border-style:none;
border:1px solid #B4C2D7;
color:#324C6C;
padding:5px;
min-height:20px;
border-radius:5px;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #C0D3EB,stop:1 #BCCFE7);
}

.QPushButton:hover{
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #D2E3F5,stop:1 #CADDF3);
}

.QPushButton:pressed{
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #C0D3EB,stop:1 #BCCFE7);
}

.QPushButton:disabled{
color:#838383;
background:#F4F4F4;
}

QPushButton#btnSplitterH{
padding:2px;
min-height:8px;
}

QPushButton#btnMenu,QPushButton#btnMenu_Min,QPushButton#btnMenu_Max,QPushButton#btnMenu_Close,QPushButton#btnSplitterV,QPushButton#btnSplitterH{
border-radius:0px;
color:#324C6C;
background-color:rgba(0,0,0,0);
border-style:none;
}

QPushButton#btnMenu:hover,QPushButton#btnMenu_Min:hover,QPushButton#btnMenu_Max:hover,QPushButton#btnSplitterV:hover,QPushButton#btnSplitterH:hover{
background-color:qlineargradient(spread:pad,x1:0,y1:1,x2:0,y2:0,stop:0 rgba(25,134,199,0),stop:1 #D2E3F5);
}

QPushButton#btnMenu_Close:hover{
background-color:qlineargradient(spread:pad,x1:0,y1:1,x2:0,y2:0,stop:0 rgba(238,0,0,128),stop:1 rgba(238,44,44,255));
}

QCheckBox{
color:#324C6C;
spacing:2px;
}

QCheckBox::indicator{
width:20px;
height:20px;
}

QCheckBox::indicator:unchecked{
image:url(:/image/checkbox_unchecked.png);
}

QCheckBox::indicator:checked{
image:url(:/image/checkbox_checked.png);
}

QRadioButton{
color:#324C6C;
spacing:2px;
}

QRadioButton::indicator{
width:15px;
height:15px;
}

QRadioButton::indicator::unchecked{
image:url(:/image/radio_normal.png);
}

QRadioButton::indicator::checked{
image:url(:/image/radio_selected.png);
}

QSpinBox::up-button,QDateEdit::up-button,QTimeEdit::up-button,QDateTimeEdit::up-button{
image:url(:/image/add_top.png);
}

QSpinBox::down-button,QDateEdit::down-button,QTimeEdit::down-button,QDateTimeEdit::down-button{
image:url(:/image/add_bottom.png);
}

QComboBox,QDateEdit,QTimeEdit,QDateTimeEdit,QSpinBox{
border-radius:3px;
padding:3px 5px 3px 5px;
border:1px solid #B4C2D7;
background:none;
selection-background-color:#C0D3EB;
selection-color:#324C6C;
}

QComboBox::drop-down,QDateEdit::drop-down,QTimeEdit::drop-down,QDateTimeEdit::drop-down{
subcontrol-origin:padding;
subcontrol-position:top right;
width:15px;
border-left-width:1px;
border-left-style:solid;
border-top-right-radius:3px;
border-bottom-right-radius:3px;
border-left-color:#B4C2D7;
}

QComboBox::down-arrow,QDateEdit::down-arrow,QTimeEdit::down-arrow,QDateTimeEdit::down-arrow{
image:url(:/image/add_bottom.png);
}

QMenu{
color:#324C6C;
background-color:#C0D3EB;
margin:2px;
}

QMenu::item{
padding:3px 20px 3px 20px;
}

QMenu::indicator{
width:13px;
height:13px;
}

QMenu::item:selected{
color:#324C6C;
border:0px solid #B4C2D7;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #C0D3EB,stop:1 #BCCFE7);
}

QMenu::separator{
height:1px;
background:#B4C2D7;
}

QProgressBar{
background:#BCCFE7;
border-radius:5px;
text-align:center;
border:1px solid #C0D3EB;
}

QProgressBar::chunk{
width:5px;
margin:0.5px;
background-color:#D2E3F5;
}

QSlider::groove:horizontal,QSlider::add-page:horizontal{
height:8px;
border-radius:3px;
background:#BCCFE7;
}

QSlider::sub-page:horizontal{
height:8px;
border-radius:3px;
background:#CADDF3;
}

QSlider::handle:horizontal{
width:13px;
margin-top:-3px;
margin-bottom:-3px;
border-radius:6px;
background:qradialgradient(spread:pad,cx:0.5,cy:0.5,radius:0.5,fx:0.5,fy:0.5,stop:0.6 #FFFFFF,stop:0.8 #BCCFE7);
}

QSlider::groove:vertical,QSlider::sub-page:vertical{
width:8px;
border-radius:3px;
background:#BCCFE7;
}

QSlider::add-page:vertical{
width:8px;
border-radius:3px;
background:#CADDF3;
}

QSlider::handle:vertical{
height:13px;
margin-left:-2px;
margin-right:-3px;
border-radius:6px;
background:qradialgradient(spread:pad,cx:0.5,cy:0.5,radius:0.5,fx:0.5,fy:0.5,stop:0.6 #FFFFFF,stop:0.8 #BCCFE7);
}

QScrollBar:vertical{
width:10px;
background-color:rgba(0,0,0,0%);
padding-top:10px;
padding-bottom:10px;
}

QScrollBar:horizontal{
height:10px;
background-color:rgba(0,0,0,0%);
padding-left:10px;
padding-right:10px;
}

QScrollBar::handle:vertical,QScrollBar::handle:horizontal{
width:10px;
background:#CADDF3;
}

QScrollBar::handle:vertical:hover,QScrollBar::handle:horizontal:hover{
width:10px;
background:#CADDF3;
}

QScrollBar::add-line:vertical{
height:10px;
width:10px;
subcontrol-position:bottom;
subcontrol-origin:margin;
border-image:url(:/image/add_bottom.png);
}

QScrollBar::add-line:horizontal{
height:10px;
width:10px;
subcontrol-position:right;
subcontrol-origin:margin;
border-image:url(:/image/add_right.png);
}

QScrollBar::sub-line:vertical{
height:10px;
width:10px;
subcontrol-position:top;
subcontrol-origin:margin;
border-image:url(:/image/add_top.png);
}

QScrollBar::sub-line:horizontal{
height:10px;
width:10px;
subcontrol-position:left;
subcontrol-origin:margin;
border-image:url(:/image/add_left.png);
}

QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical,QScrollBar::add-page:horizontal,QScrollBar::sub-page:horizontal{
width:10px;
background:#BCCFE7;
}

QScrollArea{
border:0px;
}

QTreeView,QListView,QTableView,QTabWidget::pane{
border:1px solid #B4C2D7;
selection-background-color:#D2E3F5;
selection-color:#324C6C;
alternate-background-color:#CADDF3;
}

QTableView::item:selected,QListView::item:selected,QTreeView::item:selected{
color:#324C6C;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #C0D3EB,stop:1 #BCCFE7);
}

QTableView::item:hover,QListView::item:hover,QTreeView::item:hover{
color:#324C6C;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #D2E3F5,stop:1 #CADDF3);
}

QTableView::item,QListView::item,QTreeView::item{
padding:5px;
margin:0px;
}

QHeaderView::section,QTableCornerButton:section{
padding:3px;
margin:0px;
color:#324C6C;
border:1px solid #B4C2D7;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #D2E3F5,stop:1 #CADDF3);
}

QTabBar::tab{
border-radius:5px;
border:1px solid #B4C2D7;
color:#324C6C;
min-width:55px;
min-height:20px;
padding:3px 8px 3px 8px;
margin:1px;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #D2E3F5,stop:1 #CADDF3);
}

QTabBar::tab:selected,QTabBar::tab:hover{
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #C0D3EB,stop:1 #BCCFE7);
}

QStatusBar::item{
border:0px solid #C0D3EB;
border-radius:3px;
}

QToolBox::tab,QToolTip,QGroupBox#gboxDevicePanel{
padding:3px;
border-radius: 5px;
color:#324C6C;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #C0D3EB,stop:1 #BCCFE7);
}

QToolBox::tab:selected{
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #D2E3F5,stop:1 #CADDF3);
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="currentStatusLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>450</y>
      <width>371</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>等待输入服务器信息</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>221</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>服务器信息</string>
    </property>
    <widget class="QLineEdit" name="portLineEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>IP地址：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="hostLineEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>端口：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="connectButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>75</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>建立连接</string>
     </property>
    </widget>
    <widget class="QPushButton" name="disconnectButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>75</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>断开连接</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>221</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>文件传输</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>进度：</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="clientProgressBar">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QPushButton" name="openButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>75</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>打开文件</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sendFileButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>75</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>发送文件</string>
     </property>
    </widget>
    <widget class="QPushButton" name="openFolderButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>121</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>打开接收文件夹</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>381</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>信息显示</string>
    </property>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>361</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="sendMesButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>440</y>
      <width>75</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>440</y>
      <width>75</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>310</y>
      <width>381</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>信息输入</string>
    </property>
    <widget class="QTextEdit" name="inputTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>361</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="helpButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>440</y>
      <width>75</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>帮助</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
